statistics/handle/handle.go

```diff
@@ -654,3 +673,20 @@ func (h *Handle) columnCountFromStorage(tableID, colID int64) (int64, error) {
 	}
 	return rows[0].GetMyDecimal(0).ToInt()
 }
+
+func (h *Handle) statsMetaByTableIDFromStorage(tableID int64, historyStatsExec sqlexec.RestrictedSQLExecutor) (version uint64, modifyCount, count int64, err error) {
```

Comment:
seems table id is the only way to get stats meta from storage? Can we simplify this function name to `statsMetaFromStorage()`?

Reply:
We can also `select * from mysql.stats_meta` without any `where-conditions`.